.github/container/test-jax.sh

```diff
@@ -113,8 +113,6 @@ for t in $*; do
 done
 
 TEST_TAG_FILTER_ARRAY=()
-TEST_TAG_FILTER_ARRAY+=('-multiaccelerator')
```

Comment:
Doesn't that remove tests?

Reply:
I believe that before this PR we were *not* running the `multiaccelerator` tests (because of the minus), and with this PR we *are* running them.